overlord/devicestate/devicestate_systems_test.go

```diff
@@ -335,40 +335,38 @@ func (s *deviceMgrSystemsSuite) TestRequestSameModeSameSystem(c *C) {
 	s.state.Unlock()
 
 	label := s.mockedSystemSeeds[0].label
-
-	happyModes := []string{"run", "recover"}
```

Comment:
shouldn't we have just moved recover to sadModes, are we not testing that?

Reply:
Yeah, silly me. I'll restore the test.